src/main/java/com/mulberry/blackmulberry/exception/DuplicateResourceException.java

```diff
@@ -0,0 +1,23 @@
+package com.mulberry.blackmulberry.exception;
+
+public class DuplicateResourceException extends Exception {
```

Comment:
is it okay that your exception is checked?

Reply:
can you tell me in more detail what i did wrong?